syntax = "proto3";
option go_package="./;protocol";

package protocol;

// 28 Bytes Header
message Packet{
  fixed64 seq = 1;
  fixed64 timestamp = 2;
  fixed32 cmd = 3;
  fixed32 msg_type = 4;
  fixed32 data_length = 5;
  bytes data = 6;
}
// 消息结构
message Msg{
  uint64 msg_id = 1;
  uint64 conversation_id = 2;
  uint64 sender_id = 3;
  uint64 receiver_id = 4;
  int32 content_type = 5;
  bool is_group = 6;
  bytes content = 7;
}